modules/oc-mirror-imageset-config-parameters-v2.adoc

```diff
@@ -35,6 +35,23 @@ Example: `mirror.openshift.io/v2alpha1`
 |Integer 
 Example: `4`
 
+|`kubeVirtContainer`
+|When `true`, includes images from the HyperShift KubeVirt CoreOS container.
```

Comment:
@subhtk  I think we should say like `when true includes images from release payload for HyperShift KubeVirt CoreOS container` ?

@sherine-k WDYT ?

Reply:
sounds good!